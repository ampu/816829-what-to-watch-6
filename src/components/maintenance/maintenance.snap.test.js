import React from 'react';

import {render} from '@testing-library/react';

import Maintenance from './maintenance';

describe(`Maintenance component`, () => {
  it(`Should render according to the snapshot`, () => {
    const {container} = render(<Maintenance/>);
    expect(container).toMatchSnapshot();
  });

  it(`Should render according to the snapshot when isInverse equals to true`, () => {
    const {container} = render(<Maintenance isInverse/>);
    expect(container).toMatchSnapshot();
  });

  it(`Should render according to the snapshot when children are provided`, () => {
    const {container} = render(<Maintenance>Hello World!</Maintenance>);
    expect(container).toMatchSnapshot();
  });
});
