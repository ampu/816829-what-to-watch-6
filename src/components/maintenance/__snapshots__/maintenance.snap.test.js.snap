// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Maintenance component Should render according to the snapshot 1`] = `
<div>
  <div
    class="maintenance"
  >
    Maintenance, sorry...
  </div>
</div>
`;

exports[`Maintenance component Should render according to the snapshot when children are provided 1`] = `
<div>
  <div
    class="maintenance"
  >
    Hello World!
  </div>
</div>
`;

exports[`Maintenance component Should render according to the snapshot when isInverse equals to true 1`] = `
<div>
  <div
    class="maintenance maintenance--inverse"
  >
    Maintenance, sorry...
  </div>
</div>
`;
